File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Delegate/DelegateGenerics.vb
Imports System.Collections.Generic

Public Delegate Sub SingleGenericType(Of T)()

Public Delegate Sub MultipleGenericType(Of T As {Class, IList(Of String)}, Y As Structure)()

Public Delegate Sub CovariantGenericType(Of In T)()

Public Delegate Sub ContravariantGenericType(Of Out T)()

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Delegate/DelegateGenerics.cs
using System.Collections.Generic;

public delegate void SingleGenericType<T>();

public delegate void MultipleGenericType<T, Y>()
    where T : class, IList<string>
    where Y : struct;

public delegate void CovariantGenericType<in T>();

public delegate void ContravariantGenericType<out T>();